src/Mantasflowers.WebApi/Middleware/GlobalExceptionHandlingMiddleware.cs

```diff
@@ -75,6 +75,9 @@ private Task HandleGlobalException(Exception e, HttpContext context)
                 case UserNotFoundException _:
                     statusCode = HttpStatusCode.NotFound;
                     break;
+                case HashMapOrderNotFoundException _:
+                    statusCode = HttpStatusCode.NotFound;
+                    break;
```

Comment:
Since it doesn't require any special handling beyond setting status code, you can have something like this:
```csharp
// ...
case FirebaseUidNotFoundException _:
case UserNotFoundException _:
case HashMapOrderNotFoundException _:
    statusCode = HttpStatusCode.NotFound;
    break;
// ...
```